modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

```diff
@@ -24,7 +24,13 @@ This step can cause delays in running applications.
 . Copy Runtime Manager Agent Configuration files
 . ZIP all the information
 
-After the script is executed, you can find a .zip file in the folder `$MULE_HOME/bin` (with a name similar to `snapshot_201701021410.zip`) that you can send to MuleSoft's support engineers to help troubleshoot your issues.
+After the script completes, you can find a .zip file in the folder `$MULE_HOME/bin` (with a name similar to `snapshot_201701021410.zip`) that you can send to MuleSoft's support engineers to help troubleshoot your issues.
```

Comment:
The MuleSoft style guide says ZIP not .zip (unless it's actually the name of the file).